src/resources/Restoration/BC.ts

```diff
@@ -22,7 +22,6 @@ export const RestorationResourceBc: RestorationResourceIF = {
     header: 'Add Applicant Information',
     blurb: null,
     helpSection: null,
-    addOrganization: 'Add a Business or a Corporation',
```

Comment:
This is the same for all restorations so it is now a static string in the Vue component itself.